app/models/jubla/role.rb

```diff
@@ -143,8 +143,10 @@ def alumnus_manager_destroy_for_alumnus
   end
 
   def alumnus_manager
-    @alumnus_manager ||= Jubla::Role::AlumnusManager.new(
-      self, skip_alumnus_callback: skip_alumnus_callback
-    )
+    @alumnus_manager ||= if %w[Flock ChildGroup FlockAlumnusGroup].include? type.split("::", 3)[1]
```

Comment:
```suggestion
    @alumnus_manager ||= if %w[Flock ChildGroup FlockAlumnusGroup].include? type.demodulize
```
Wenn Nichts dagegen spricht wäre das noch ein wenig eleganter

Reply:
Leider brauche ich das zweite von drei Segementen. `type` ist zum Beispiel `Group::Flock::Leader`. Ich hatte aber kurz überlegt, ob ich refinement auf String machen, damit das einen besseren Namen bekommt. Oder noch eine Methode auf Role, die z.B. `flocky?` oder so heisst und diese Prüfung dann schön lesbar kapselt. Beides erschient mir dann aber doch etwas Overkill.

Reply:
Ah, I see. :+1: